package.json

```diff
@@ -30,14 +35,18 @@
   },
   "homepage": "https://github.com/commercetools/commercetools-adyen-integration#readme",
   "devDependencies": {
+    "adyen-cse-js": "git+https://github.com/Adyen/adyen-cse-web.git#0_1_23",
```

Comment:
it says it's deprecated
https://github.com/Adyen/adyen-cse-web#deprecated-adyen-cse-for-web

Reply:
It is, but this is the only way I found for automated testing without frontend.

Reply:
And when I wrote the test, it was not deprecated :( https://github.com/Adyen/adyen-cse-web/tree/a5cf29e75cf69874c9af96258bbfa0bcd6729ce0

Reply:
For now I will leave it here, I already made a fork of this branch. Later the tests need to be changed and this dependency needs to be removed. https://github.com/commercetools/commercetools-adyen-integration/issues/22